components/discord_webhook/actions/send-message-common.mjs

```diff
@@ -39,6 +39,14 @@ export default {
     },
   },
   methods: {
+    appendPipedreamText(message) {
```

Comment:
I like moving the logic of this function to the common file so it can be reused 👍.